integration-libs/digital-payments/src/checkout/dp-checkout.module.ts

```diff
@@ -28,6 +29,7 @@ import { DpPaymentMethodModule } from './cms-components/dp-payment-method/dp-pay
       useExisting: OccDpRequestNormalizer,
       multi: true,
     },
+    DpCheckoutPaymentService,
```

Comment:
A question: what is the reason for providing the service in this module? it seems it's already provided in `root` and used withing the DP lib?

Reply:
Without it, there will be `NullInjectorError: No provider for DigitalPaymentsAdapter!`.

Reply:
`CartConnector` also has `providedIn: 'root'`, but if not put it in module's provider list, there will be "No provier error".